src/modules/parser.ts

```diff
@@ -260,9 +266,10 @@ export function parse (query: QueryType, type: GotQL.ExecutionType): string {
     if (!query.operation) throw new Error('a query must have at least one operation')
     if (!type) throw new Error('type must be either "query" or "mutation"')
 
+    let isMutation = type === 'mutation'
```

Comment:
This is not needed. There's a type called `GotQL.ExecutionType` which is an `enum` with two values: MUTATION and QUERY.

```suggestion

```

Reply:
good catch